bgpd/bgp_routemap.c

```diff
@@ -365,38 +365,43 @@ static enum route_map_cmd_result_t
 route_match_script(void *rule, const struct prefix *prefix, void *object)
 {
 	const char *scriptname = rule;
+	const char *routematch_function = "route_match";
 	struct bgp_path_info *path = (struct bgp_path_info *)object;
 
-	struct frrscript *fs = frrscript_load(scriptname, NULL);
+	struct frrscript *fs = frrscript_new(scriptname);
 
-	if (!fs) {
-		zlog_err("Issue loading script rule; defaulting to no match");
+	if (frrscript_load(fs, routematch_function, NULL)) {
+		zlog_err(
+			"Issue loading script or function function; defaulting to no match");
 		return RMAP_NOMATCH;
 	}
 
-	enum frrlua_rm_status lrm_status = LUA_RM_FAILURE,
+	enum frrlua_rm_status status_failure = LUA_RM_FAILURE,
```

Comment:
previously, `lrm_status` was represented the failure status as well as the action taken, but since now we're getting the action back via `frrscript_get_result`, lrm_status now just represents failure status

Reply:
Actually, I figured I could pass in LUA_RM_FAILURE etc. directly. https://github.com/FRRouting/frr/pull/8982/commits/59a35b667d4f592f284e7c13db9123b7eb6379b6